src/reducers/GameState.js

```diff
@@ -24,7 +24,7 @@ const playGame = state => ({ ...state, gameState: STATES.PLAYING })
 const changeActionSelected = (state, { actionType }) => ({
   ...state,
   actionSelected: actionType,
-  machineSelected: actionType !== SELECTION ? null : state.machineSelected,
+  machineSelected: EDITIONS.includes(actionType) ? null : state.machineSelected,
```

Comment:
the idea is that you never has a machine selected and  actionSelected in the same time, for that this check ` actionType !== SELECTION `

Reply:
Testing it, i look that is working fine, forgot my comment

Reply:
ok!!